index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MERCY MUGAMBI</title>
+
+    <!--stylesheet-->
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <header class="header-nav">
+      <div class="logo">MER<span>cy</span></div>
+
```

Comment:
- [ ] Kindly wrap your logo in an `<a>` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info